docs/architecture/adr-0001-app-setup-and-goals.md

```diff
@@ -0,0 +1,168 @@
+# Context
+
+We need to create an application without a dependency on Raster Foundry that has
+the following capabilities:
+
+- run models on imagery
+- kick off model runs when it learns about new imagery
+- run models without knowing anything about the content of those models
+- summarize past model results
+- allow users to respond to model run events
+
+Additionally, we'd like the application to meet several design goals that aren't
+core to the pursuit of those features:
+
+- we should deploy continuously
+- infrastructure should be kept as light as possible
+
+These requirements are located roughly a third of the way from "core competency
+team improvement" to vanity.
+
+The main question this ADR seeks to answer is how the API works and how
+communication between the external world and the API and the API and the model
+runner will work. This ADR will not attempt to answer questions of what a user interface
+should accomplish, since that is not germane to the short-term (three month-ish)
+uses we envision for this project.
+
+## Specific Components
+
+### Summary
+
+TODO
+
+### New imagery from the outside world
+
+When new imagery becomes available from the outside world, this application will
+need to match it against models that are interested in that kind of imagery. This
+requires tracking some kind of metadata about the incoming imagery. We
+developed a fairly complex datamodel in Raster Foundry that tracks information about
+imagery, including component files for that imagery and band-level information.
+However, we are not a spec, and to the best of our knowledge no one relies on the
+Raster Foundry datamodel to describe what imagery updates look like.
+
+Instead, the API will demand that incoming imagery adheres to the [STAC electro-optical](https://github.com/radiantearth/stac-spec/tree/master/extensions/eo)
+spec extension. There are several reasons to prefer using STAC for a datamodel.
+These include tapping into ongoing work in the geospatial community to standardize
+data exchange, making validation easier for anyone hoping to rely on this API,
+and reusing our own work around a type-safe STAC datamodel in `geotrellis-server`.
+This choice also commits us to using Scala as the backend language. Using anything
```

Comment:
The focus on the prior decision necessitating a backend in Scala feels out-of-place here. It doesn't seem particularly valuable within the context of the broader ADR. I would consider omitting this sentence.